roles/elastic/filter_plugins/custom.py

```diff
@@ -0,0 +1,75 @@
+__author__ = "dale mcdiarmid"
+
+import re
+import os.path
+from six import string_types
+
```

Comment:
The python bytecode (*.pyc) file should be git-ignored (so does the `__pycache__` directory).